clang/test/CodeGen/instantiate-new-placement-size.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: %clang++ -S -emit-llvm -o - %s | FileCheck %s
+#include <new>
+
+// CHECK: call void @llvm.memset.p0.i64(ptr align 1 %x, i8 0, i64 8, i1 false)
+// CHECK: call void @llvm.memset.p0.i64(ptr align 16 %x, i8 0, i64 32, i1 false)
+template <typename TYPE>
```

Comment:
We need more test cases to make sure we are not breaking other cases. Non-dependent Vs dependent, constant Vs dynamically sized, VLA etc